src/js/components/campaignpage/Charities.jsx

```diff
@@ -43,11 +44,45 @@ challenges facing our planet."`,
  * 
  * @param {!NGO[]} charities
  * @param {{string:Money}} donation4charity - charity ID to donation amount
+ * @param {?Boolean} filterLowDonations If true low donation charities will not display
+ * @param {?Number} lowDonationThreshold Custom threshold number to treat charities as "low donation". 1% of the total by default
+ * @param {?Boolean} showLowDonations If false will hide the donation number for low donation charities
+ * @param {?Boolean} showDonations If false will hide all donation numbers
+ * @param {?String[]} hideCharities hide specific charities by ID
  */
-const Charities = ({ charities, donation4charity }) => {
+const Charities = ({ charities, donation4charity, campaignPage }) => {
 	
+	//filterLowDonations, lowDonationThreshold, showLowDonations, showDonations, hideCharities
+	let {lowDntnDisplay, lowDntn, hideCharities} = campaignPage;
```

Comment:
Are these part of the CampaignPage object?

Reply:
Yes - comments added